### DISCUSSION TITLE
Any way to create a auto height for the BottomSheet to fit its content?

### DISCUSSION THREAD

**Discussion by hellocaio** [gorhom/react-native-bottom-sheet#1130] [Q&A] [answered]
Hey guys,

Love this project. I'm trying build a auto-height prop that resizes the bottom sheet to fit the content inside of it. I've been attempting to use `onLayout` prop to measure the height of its content then set it dynamically to the state of the BottomSheet component. It works great for anything other than a FlatList. The `onLayout` callback is called for every item in the FlatList, this causes the Bottom Sheet height to change with the render of every item in the list, causing a shrinking/expanding behavior of the bottom sheet.

Does anyone have any recommendation on how to tackle this?

Thanks!

**Comment by Eli-Nathan** [comment, marked as answer]
Upgrade to [4.5.0](https://github.com/gorhom/react-native-bottom-sheet/releases/tag/v4.5.0) This was just added 😄
